EIPS/eip-2212.md

```diff
@@ -0,0 +1,271 @@
+
+We assume that most apps don't need this and the spec as it is now is sufficient for running the very first experiments. Yet, time will tell best. We are looking forward to making updates if need be.
+
+#### Parallel Staking
+
+Large enterprises have multiple sources of revenue, hence switching to a business model based on staking would require a smart contract that accepts multiple stakes per user.
+
+However, the goal is to avoid writing an over-optimised standard. Parallel staking would require dapp creators to keep track of multiple stake ids, instead of only the user's account. Also, the user's interface would become more complex, as they would have to be aware of multiple stakes.
+
+#### Beyond Compound
+
+Some functions in the interface are highly specific to Compound, but the rationale behind that is simple. Compound is by far the best suited protocol for interest earning stakes. It is ultra-short term, which makes for frictionless business. We may update the interface when other continuous lending schemes, such as the Dai Savings Rate, will scale significantly.
+
+---
+
+### Concerns
+
+#### Loan Defaults
+
+Lending never comes without risks, so interest earning stakes are not a good fit for the ultra-sensitive business applications. That said, dapp creators have an incentive to monitor the market for borrowers that go under the collateral requirements and liquidate them. By doing so, they secure the network for their own users and also earn a fee in the process.
+
+#### Fluctuating Rates
+
+The market rates for lending stablecoins hovered around 10% in early 2019, but this is not a given. Dapp creators can accommodate this issue by asking for a stake that generates interest reasonably above what they deem as an acceptable revenue stream.
+
+#### Rent-Seeking
+
+Interest earning stakes work best for products or services that are priced on time instead of quantity (imagine subscription-based SaaS). If a dapp is quota-based, its creators would be better off charging a fee on every transfer.
+
+## Implementation
+
+[Head to the Sablier organisation on GitHub](https://github.com/sablierhq/erc2212) for a WIP implementation.
+
+## Additional References
+
+- [Thread on cTokens](https://twitter.com/PaulRBerg/status/1147980107740524544)
+- [Compound Protocol](https://github.com/compound-finance/compound-protocol)
+- [PoolTogether](https://pooltogether.us)
+- [Pooled cDAI](https://github.com/ZeframLou/pooled-cdai)
+- [Lending Proxy by Luciano Bertenasco](https://github.com/lbertenasco/lending-proxy)
+- [Web3 Revenue Primitives](https://github.com/FEMBusinessModelsRing/web3_revenue_primitives)
+
+## Copyright
+
+Copyright and related rights waived via [CC0](https://creativecommons.org/publicdomain/zero/1.0/).
+
+@PaulRBerg and @SablierHQ.
```

Comment:
What are these for?

Reply:
I saw the copyright in other standards and wrote one here too. I'll remove if its place is not here.

Reply:
There is no copyright with CC0, if I understand it correctly.

Reply:
That is why it is "waived" - it's a small thing, happy to remove it if you think it is superfluous.

Reply:
@MicahZoltu @lightclient any comments here?